### ISSUE TITLE
An error occurred during installation 

### ISSUE THREAD

**Issue by richa0892** [elastic/windows-installers#399]
This is very unfortunate and not something we'd hope to repeat.  Please let us know by filing an issue on our GitHub issues page.

=== Verbose logging started: 30-01-2021  21.40.27  Build type: SHIP UNICODE 5.00.10011.00  Calling process: C:\WINDOWS\System32\msiexec.exe ===
MSI (c) (84:30) [21:40:28:047]: Font created.  Charset: Req=0, Ret=0, Font: Req=MS Shell Dlg, Ret=MS Shell Dlg

MSI (c) (84:30) [21:40:28:048]: Font created.  Charset: Req=0, Ret=0, Font: Req=MS Shell Dlg, Ret=MS Shell Dlg

MSI (c) (84:40) [21:40:28:257]: Resetting cached policy values
MSI (c) (84:40) [21:40:28:257]: Machine policy value 'Debug' is 0
MSI (c) (84:40) [21:40:28:257]: ******* RunEngine:
           ******* Product: C:\Users\DELL 5567\Downloads\elasticsearch-7.10.1 (1).msi

**Comment by ygel** [comment]
Hello @richa0892 , once we have the fix for 7.11 #402 released, please run the installer again like so:
```
msiexec /i "<full path to msi file>" /l!*vx "<full path to log file to be created>"
```
and attach the created log file to your comment, thank you!

**Comment by pipja** [comment]
> Hello @richa0892 , once we have the fix for 7.11 #402 released, please run the installer again like so:
> 
> ```
> msiexec /i "<full path to msi file>" /l!*vx "<full path to log file to be created>"
> ```
> 
> and attach the created log file to your comment, thank you!

Still report errors with 7.11 even with the command unfortunately.

**Comment by ygel** [comment]
Hello @pipja, release containing the fix hasn't happened yet. It would be very helpful once you try again to attach full log files, as well as messages from the Windows Event Log, as outlined in the [Troubleshooting hints](https://github.com/elastic/windows-installers#where-to-look-for-information-that-helps-us-troubleshoot).
